static/src/javascripts/projects/admin/modules/abtests/abtest-report-item.js

```diff
@@ -10,7 +10,7 @@ import bonzo from 'bonzo';
 import debounce from 'lodash/functions/debounce';
 
 class ABTestReportItem extends Component {
-    constructor(config: any): void {
+    constructor(config: Object): void {
```

Comment:
I have so far only seen `(config: {})` style used for object types. I'm interested if there is a difference? 🤔